internal/observability/call.go

```diff
@@ -222,7 +221,8 @@ func (c call) endLogs(
 		return
 	}
 
-	fields := c.fields[:0]
+	backendFields := [10]zapcore.Field{}
+	fields := backendFields[:0]
```

Comment:
nit: group into var
```
var (
    tmp [10]zapcore.Field

    fields = tmp[:0]
)
```

Reply:
It seems this is not needed. Doing `make([]zap.Field, 0, 10)` still does stack allocation.